### PR TITLE
update name and links

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
@radicaled or @devoncarew ?

**Comment by devoncarew** [comment]
lgtm!

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
-# dart-lang-experimental
+# Dart plugin for Atom
 
-An experimental Dart plugin for Atom.
+An experimental [Dart](https://www.dartlang.org) plugin for [Atom](https://atom.io).
```
Comment (radicaled) [inline README.md]: Should note that this plugin is written in Dart

Comment (radicaled) [inline README.md]: Hence the "experimental" bit.

**Comment by sethladd** [comment]
Thanks!

**Pull request merged**